modules/javafx.graphics/src/main/java/com/sun/prism/es2/X11GLFactory.java

```diff
@@ -43,6 +43,7 @@
         new GLGPUInfo("advanced micro devices", null),
         new GLGPUInfo("ati", null),
         new GLGPUInfo("intel open source technology center", null),
+        new GLGPUInfo("intel", null),
```

Comment:
Since the qualification check is `vendor.startsWith(gi.vendor)` you can replace the previous `intel open source technology center"` entry with the new one.